### PR TITLE
Try to find the actual declared FMT length

### PR DESCRIPTION
**Pull request by Georacer**
Addresses #25

### CODE REVIEW

File: Ardupilog.m [hunk 1]
```diff
@@ -246,6 +249,21 @@ function delete(obj)
             % Store msgData correctly in that LogMsgGroup
             obj.(FMTName).storeData(data);
         end
+        
+        function [] = findFMTLength(obj,allHeaderCandidates)
+            for index = allHeaderCandidates
+            % Try to find the length of the format message
+                msgId = obj.log_data(index+2); % Get the next expected msgId
+                    if obj.log_data(index+3)==obj.FMTID % Check if this is the definition of the FMT message
```
Comment (hunt0r) [inline Ardupilog.m]: In this code section, I see strange indentation and a "return" call before the "warning" call. Is this intentional?

**Comment by Georacer** [comment]
@hunt0r 
I updated this branch because it had merge conflicts. I needed to re-write its history to do that.
Can you review this (and merge if it's ok) before it gets stale?

Thanks

**Comment by hunt0r** [comment]
Can you see my inline comment? I tried to start a "code review" for something that looked suspicious... but maybe I didn't submit it?

**Review by hunt0r — COMMENTED** [review]
I tried to put a comment in my code review to flag something that looks suspicious. Maybe it doesn't show till I submit this comment?

**Comment by Georacer** [comment]
I actually had 2 misplaced returns. Thanks for the review.

**Pull request merged**